Editors/BackgroundEditor.cpp

```diff
@@ -76,11 +76,11 @@ void BackgroundEditor::on_actionLoadImage_triggered() {
 
   if (dialog->exec() && dialog->selectedFiles().size() > 0) {
     QString fName = dialog->selectedFiles()[0];
-    if (fName.endsWith("Background.gmx")) {
-      Background* bkg = gmx::LoadBackground(fName.toStdString());
-      if (bkg != nullptr) {
+    if (fName.endsWith("Background.gmx") || fName.endsWith(".bkg")) {

```

Comment:
Are these checks case-insensitive? The capital B suggests not.

Reply:
these checks need work but jus tryna fix build here